src/emailLogin/removeGithubAccess.js

```diff
@@ -0,0 +1,52 @@
+const axios = require('axios');
+const fs = require('fs');
+
+const { Octokit } = require('octokit');
+
+// name of repo
+const REPO = process.argv[2];
+
+const { PERSONAL_ACCESS_TOKEN, GITHUB_ORG_NAME } = process.env;
+
+const octokit = new Octokit({
+  auth: PERSONAL_ACCESS_TOKEN,
+});
```

Comment:
#55 seems to have changed this to `GITHUB_ACCESS_TOKEN` fyi; also doesn't the token itself need to have sufficient perms to do the operations below? if so, a personal token might not cut it and i'd be in favour of calling it `GITHUB_ACCESS_TOKEN` or smth along those lines to convey the elevated perms required

Reply:
Sure, changed in https://github.com/isomerpages/isomer-conversion-scripts/pull/56/commits/39c30003fe4a7c539d1264d016f22e0c0d6b7360